core/src/main/java/io/fabric8/maven/core/service/kubernetes/KubernetesBuildService.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2016 Red Hat, Inc.
+ *
+ * Red Hat licenses this file to you under the Apache License, version
+ * 2.0 (the "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package io.fabric8.maven.core.service.kubernetes;
+
+import io.fabric8.kubernetes.client.KubernetesClient;
+import io.fabric8.maven.core.service.BuildService;
+import io.fabric8.maven.core.service.Fabric8ServiceException;
+import io.fabric8.maven.docker.config.ImageConfiguration;
+import io.fabric8.maven.docker.service.ServiceHub;
+import io.fabric8.maven.docker.util.Logger;
+
+/**
+ * @author nicola
+ * @since 17/02/2017
+ */
+public class KubernetesBuildService implements BuildService {
```

Comment:
Thats probably better called `DockerBuildService`, since it has nothing to do with Kubernetest. Building is done against a Docker daemon.